lib/msal-common/src/client/BaseClient.ts

```diff
@@ -65,28 +64,7 @@ export abstract class BaseClient {
         this.networkClient = this.config.networkInterface;
 
         // Default authority instance.
-        this.defaultAuthorityInstance = AuthorityFactory.createInstance(
```

Comment:
Will this not break msal-browser code?

Reply:
SPAClient was also updated - it's part of this PR.